lock_unlock_feature.ino

```diff
@@ -0,0 +1,70 @@
+#include <Keyboard.h>
+#include <SPI.h>
+#include <MFRC522.h>
+
+#define SS_PIN 10
+#define RST_PIN 5
+#define KEY_RETURN 0xB0  // The hex value for the return key is 0xB0.
+
+MFRC522 mfrc522(SS_PIN, RST_PIN);
+char Enter = KEY_RETURN;  // Return key is declared as Enter.
+String readid;
+String card1 = "3A3D9D9E";  // Change this value to the UID of your card.
```

Comment:
In future, this should not be a string. This would only mean one card can be used at a time. the value of this should be read from the card itself and put into an array of cards.

Reply:
Thanks for pointing out but this has been already changed.

Reply:
where is the change? I can not see it in the commits.